src/main/java/com/mazyavr/schedule/service/ProjectService.java

```diff
@@ -10,29 +14,48 @@ public class ProjectService {
 
   @Autowired
   private ProjectRepository projectRepository;
+  @Autowired
+  private UserRepository userRepository;
+
+  public Iterable<ProjectEntity> getAll(long userId) {
+    List<ProjectEntity> projects = new ArrayList<>();
+
+    for (ProjectEntity t : projectRepository.findAll()) {
+      if (t.getUser().getId() == userId) {
+        projects.add(t);
+      }
+    }
 
-  public Iterable<ProjectEntity> getAll() {
-    return projectRepository.findAll();
+    return projects;
   }
 
   public void delete(Long id) {
     projectRepository.deleteById(id);
   }
 
-  public ProjectEntity add(String name) {
-    
+  public ProjectEntity add(long userId, String name) {
+
     ProjectEntity project = new ProjectEntity();
     project.setName(name);
-    
+
+    var user0 = userRepository.findById(userId);
+
+    if (user0.isEmpty()) {
+      throw new IllegalArgumentException("No such user");
+    }
+
+    UserEntity user = user0.get();
```

Comment:
Надо определиться, как опшионалы обозначаются, либо заглавная о на конце, либо ноль. У нас до этого обозначались с о, так что предлагается продолжать эту практику, но это так лирическое отступление, не слишком важно

```suggestion
    var userO = userRepository.findById(userId);

    if (userO.isEmpty()) {
      throw new IllegalArgumentException("No such user");
    }

    UserEntity user = userO.get();
```